### PR TITLE
Request to merge

### PR DESCRIPTION
**Pull request by Gleennkar**
#Google webpage
- Add index.html and style.css
- Used fontawesome for icons
- run linters checks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    <div>
+        <Img src="Go.png">
+           <p><i class="fas fa-microphone"></i></p>
+    </div>
+    
+    <div class="flex-container">
+        <div>Google Search</div>
+        <div>I'm Feeling Lucky</div>
+      </div>
```
Comment (Gleennkar) [inline index.html:27]: You did make use of HTML semantic tags which is great but I will advise; it should be **navbar** and under this, you can then put the **ul** tag.

Comment (Gleennkar) [inline index.html:27]: After giving this **div** a class attribute, i think what should follow next should be an **h4** heading and not another **div** elements

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<body>
+    <ul>
+        <li a href="#Gmail">Gmail</li> 
+        <li a href="#Images">Images</li> 
+        <i class="fas fa-th"></i> 
+    <nav><li class="Sign In">Sign In</li></nav>
+    </ul>
+
+    <div>
+        <Img src="Go.png">
```
Comment (Gleennkar) [inline index.html:20]: always make use of the alt in the **img** tag for example **img src= "Go.png" alt="Google"** and it should be attributed a class logo which will make it easier to style and a best practice

File: style.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+li {
+    display: inline-block;
+    margin: 10px 15px;    
+}
+img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 60%;
+    padding: 70px 90px;
```
Comment (Gleennkar) [inline style.css:14]: i will suggest you reduce the font-size of the google image and flex to center the image

**Review by Gleennkar — COMMENTED** [review]
You have been doing a tremendous job so far 👌
There are some issues that you still need to work on to go to the next project but you are almost there!


Linters' error check should always indicate green ✔️

Required Changes ♻️
 Please also check inline comments under the review

**Review by aimemalaika — APPROVED** [review]
Hi @Gleennkar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**